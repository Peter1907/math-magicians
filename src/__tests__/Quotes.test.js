/* eslint-disable*/ 

import renderer from 'react-test-renderer';
import Quote from '../pages/Quotes';

describe ('Test components', () => {
    it('Quote test', () => {
        const tree = renderer.create(<Quote />).toJSON();
        expect(tree).toMatchSnapshot();
      })


});
