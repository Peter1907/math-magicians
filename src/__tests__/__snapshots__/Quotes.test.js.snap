// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test components Quote test 1`] = `
<section
  className="quotes"
>
  <p
    className="quote"
    style={
      Object {
        "fontSize": "24px",
        "paddingBottom": "20px",
      }
    }
  />
  <p
    className="author"
    style={
      Object {
        "color": "#c6e6ff",
        "fontFamily": "'Goudy Bookletter 1911', serif",
        "fontSize": "26px",
        "fontStyle": "italic",
      }
    }
  >
    _
    .
  </p>
</section>
`;
